core/migrations/0028_auto_20180905_0927.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.0.1 on 2018-09-05 09:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0027_auto_20180831_1209'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='userfeedback',
+            name='resolved',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AlterField(
+            model_name='userfeedback',
+            name='report_type',
+            field=models.CharField(choices=[('feedback', 'feedback'), ('bug', 'bug'), ('feature_request', 'feature_request')], max_length=10),
+        ),
+    ]
```

Comment:
Regenerate the migration afresh after making the changes to the model, to have only one file.